### DISCUSSION TITLE
cpp/memory/dangling_pointers

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#36] [Announcements]
# cpp/memory/dangling_pointers

编程指北: 计算机系统学习指南, 操作系统, 计算机网络, C++, Java, 算法, 数据结构

https://csguide.cn/cpp/memory/dangling_pointers.html

**Comment by YIHONG-JIN** [comment]
野指针专指未被初始化指针是否会容易理解些？

**Reply by imarvinle** [reply]
嗯嗯 似乎这样更好
